### PR TITLE
fix: navigation to linked records screen

### PR DESCRIPTION
**Pull request by IlyesBenAmara**
- this PR fixes the issue on the linked record screen

### CODE REVIEW

File: src/components/linkedRecords/LinkedRecords.tsx [hunk 1]
```diff
@@ -52,6 +52,8 @@ const LinkedRecords = () => {
     return <NotFound />
   }
 
+  console.log(uid)
```
Comment (marrouchi) [inline src/components/linkedRecords/LinkedRecords.tsx:55]:
```suggestion
  console.log(uid)
```
Please remove this

File: src/services/mockBackend.ts [hunk 2]
```diff
@@ -42,6 +42,7 @@ axiosMockAdapterInstance
   .reply(config => {
     const id = config.url?.split('/').pop()
     if (goldenRecord.uid === id) {
+      console.log(config)
```
Comment (marrouchi) [inline src/services/mockBackend.ts:45]: This as well :pray:

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: src/components/linkedRecords/LinkedRecords.tsx (hunk 1 above)
Comment (IlyesBenAmara) [reply src/components/linkedRecords/LinkedRecords.tsx:55]: my bad :sweat_smile:

**Review by marrouchi — APPROVED** [review]

**Comment by IlyesBenAmara** [comment]
I added integration changes so that we could display the linked records. @MatthewErispe @Lumiere-Lodi @marrouchi let me know what you think guys ;)

**Review by marrouchi — CHANGES_REQUESTED** [review]

**Review by marrouchi — APPROVED** [review]

**Review by Lumiere-Lodi — APPROVED** [review]
Looks good to me. At this point since we haven't updated the mockData for golden search result it will only work with the actual API and not the mock API. However, all is good.

**Review by MatthewErispe — APPROVED** [review]
LGTM

**Pull request merged**